packages/react-components/stories/lift-marker.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { LiftMarker } from '../lib';
+import { makeLift, makeLiftState } from '../tests/lifts/test-utils';
+
+export default {
+  title: 'Lift Markers',
+  component: LiftMarker,
+} as Meta;
+
+function makeStory(
+  lift: RomiCore.Lift,
+  liftState?: RomiCore.LiftState,
+  isInCurrentFloor = true,
+): Story {
+  return (args) => (
+    <svg viewBox="-2 -2 4 4" width={400} height={400}>
+      <LiftMarker lift={lift} liftState={liftState} isInCurrentFloor={isInCurrentFloor} {...args} />
+    </svg>
+  );
+}
+
+export const Basic = makeStory(makeLift(), makeLiftState());
+
+export const UnknownState = makeStory(makeLift());
+
+export const Rotated = makeStory(
+  makeLift({
+    ref_yaw: Math.PI / 4,
+    doors: [
+      {
+        name: 'door',
+        door_type: RomiCore.Door.DOOR_TYPE_DOUBLE_TELESCOPE,
+        motion_direction: 1,
+        motion_range: Math.PI / 2,
+        v1_x: -1.414,
+        v1_y: 0,
+        v2_x: 0,
+        v2_y: -1.414,
+      },
+    ],
+  }),
+  makeLiftState(),
+);
+
+// FIXME: Broken
```

Comment:
I think it's fine, this is an ongoing effort to isolate the components we are currently using, I intend to continue working on this. This PR is for a early review before we replace the current components with the ones here.

Reply:
Just an idea, since they are independent packages, and they are not imported yet, would not it be good to merge at least one base structure already?, Then new reusable components can be created there too.

Reply:
Yeah, thats the goal of this PR. If it looks good then we can merge it and start replacing the components in dashboard. I'm sure there would be issues when we actually try to replace the components which might need some refactor on the components here, they are not meant to be final so I meant for this to be an early preliminary review to see if things make sense before merging.